policy/modules/apps/screen.te

```diff
@@ -22,6 +22,13 @@ type screen_runtime_t;
 files_runtime_file(screen_runtime_t)
 ubac_constrained(screen_runtime_t)
 
+## <desc>
+##      <p>
+##      permit init to start a detached screen session
+##      </p>
+## </desc>
+gen_tunable(init_launch_screen, false)
```

Comment:
I don't think adding a tunable is necessary.

Reply:
dropped